### PR TITLE
#2 Added requirements and design

### LINKED ISSUE #2: Add requirements

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #2

### CODE REVIEW

File: doc/system_requirements.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## Stakeholders
+
+* Exasol Integration Team members
+
+## Features
+
+Features are the highest level requirements in this document that describe the main functionality of ETC.
+
+### Verify repository structure
```
Comment (AnastasiiaSergienko) [inline doc/system_requirements.md:24]: Don't forget about the header case

File: doc/system_requirements.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+`feat~verify-repo-structure~1`
+
+This plugin can verify if the repository structure fits the internal standards.
+
+There are two types of verifications:
+
+* Files that must exist (for example README.md)
```
Comment (AnastasiiaSergienko) [inline doc/system_requirements.md:31]: I think these two are requirements

File: doc/system_requirements.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+* Files that must exist and have a predefined content (for example GitHub actions)
+
+Needs: req
+
+### Fit repository structure
+`feat~fit-repo-structure~1`
+
+This plugin can create or update the required files or directories.
```
Comment (AnastasiiaSergienko) [inline doc/system_requirements.md:39]: Create / update are two separate features

File: doc/system_requirements.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+Needs: req
+
+## Requirements
+
+### Defined repository structure
+`req~repository-structure~1`
+
+This plugin defines the required structure of the repositories.
```
Comment (AnastasiiaSergienko) [inline doc/system_requirements.md:48]: The developers define the structure, not the plugin, I think. Rephrase this requirement

File: doc/system_requirements.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+For that, it provides a convenient way that can easily be adapted.
```
Comment (AnastasiiaSergienko) [inline doc/system_requirements.md:50]: This is a separate requirement

File: doc/system_requirements.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+`req~fit-repo-structure~1`
+
+This plugin provides a command that automatically creates or updates the files so that they fit 
+the required project structure.
+
+Covers:
+
+* [feat~fit-repo-structure~1](#fit-repository-structure)
+
+Needs: dsn
```
Comment (AnastasiiaSergienko) [inline doc/system_requirements.md:83]: All the checks you are planning to implement should be also documented here. Check the Release Robot's requirements as an example

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

File: doc/system_requirements.md (hunk 2 above)
Comment (jakobbraun) [reply doc/system_requirements.md:31]: so what are features?

File: doc/system_requirements.md (hunk 3 above)
Comment (jakobbraun) [reply doc/system_requirements.md:39]: But as a user, I want to have it in one feature / command. I just want to call fit and the plugin creates or updates.

File: doc/system_requirements.md (hunk 4 above)
Comment (jakobbraun) [reply doc/system_requirements.md:48]: updated

File: doc/system_requirements.md (hunk 6 above)
Comment (jakobbraun) [reply doc/system_requirements.md:83]: added

**Review by AnastasiiaSergienko — APPROVED** [review]

File: doc/design.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+The default maven lifecycle phase for this goal is the `package` phase.
+
+Covers:
+
+* `req~verify-repo-structure~1`
+
+Needs: impl, itest
+
+## Maven fit Gaol
+`dsn~mvn-fit-pahse~1`
```
Comment (AnastasiiaSergienko) [inline doc/design.md:41]: Typos

**Pull request merged**